### PR TITLE
Add Swedish translation

### PR DESCRIPTION
**Pull request by fejd**

### CODE REVIEW

**Comment by fejd** [comment]
This commit resolves #76

File: Umweltzone/src/main/res/values-sv/ckchangelog.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  Copyright (C) 2019  Tobias Preuss
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-sv/ckchangelog.xml:2]: Please change to `Copyright (C) 2020  Fredrik Henricsson`.

File: Umweltzone/src/main/res/values-sv/strings.xml [hunk 2]
```diff
@@ -0,0 +1,359 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  Copyright (C) 2020  Tobias Preuss
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-sv/strings.xml:2]: Please change to `Copyright (C) 2020  Fredrik Henricsson`.

File: Umweltzone/src/main/res/values-sv/ckchangelog.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        Overwrites ckChangeLog internal settings for all languages.
+        Will be removed when this application will be localized.
+    -->
+    <string name="changelog_title">Detta är nytt</string>
+    <string name="changelog_show_full">Mer &#8230;</string>
+
+    <string name="changelog_version_format">
+        Version <xliff:g id="version_name">%s</xliff:g>
+    </string>
+</resources>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-sv/ckchangelog.xml:28]: You are missing the translation for `changelog_error_resource_not_android_5`. See _values/ckchangelog.xml_.

File: Umweltzone/src/main/res/values-sv/ckchangelog.xml (hunk 1 above)
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-sv/ckchangelog.xml:2]: Please change to `Copyright (C) 2020  Fredrik Henricsson`.

**Review by johnjohndoe — CHANGES_REQUESTED** [review]
Hey @fejd, thank you - your translation is technical almost perfect. Please have a look at my few comments. It would be great if you could `git commit --amend` the desired changes to your commit and then `git push --force` it to update this pull request.

**Comment by fejd** [comment]
Thanks, I'll fix the comments and push an update soon!

**Review by johnjohndoe — APPROVED** [review]
Tack så mycket @fejd for the Swedish translation. I checked it and it looks great. :+1:

**Comment by johnjohndoe** [comment]
:arrow_forward: If you have time there is a few strings to translate [here](https://github.com/Umweltzone/roadsigns/issues/9).

**Pull request merged**